src/pal/src/misc/stackstring.cpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information. 
+
+template <unsigned int MEMSIZE>
+class StackString
+{
+private:
+	char m_innerBuffer[MEMSIZE];
```

Comment:
Are we OK that we will be wasting this memory when creating a StackString with size > MEMSIZE?